provider/terraform/resources/resource_dataproc_cluster.go

```diff
@@ -362,6 +362,30 @@ func instanceConfigSchema() *schema.Schema {
 					},
 				},
 
+				// Note: preemptible workers don't support accelerators
+				"accelerators": {
+					// Terraform was unhappy when this was a set, even though it probably could be one
```

Comment:
what does this mean? Did it give an error message? Is it because of the diffsuppress?